opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipEditionOverview.tsx

```diff
@@ -383,7 +395,7 @@ Omit<StixCoreRelationshipEditionOverviewProps, 'queryRef'>
                 component={DateTimePickerField}
                 name="stop_time"
                 onFocus={editor.changeFocus}
-                onSubmit={editor.changeField}
+                onSubmit={handleSubmitFieldStopTime}
```

Comment:
why doesn't it work like the other fields ? why did you need to add `handleSubmitFieldStopTime` ?

Reply:
same as for an old issue whose brainstorm is here : https://www.notion.so/filigran/Can-t-update-end-date-of-an-event-3033-ae8c2ab25c564973b974dfb3c980c90d#3e295d32596d4da391830a004964244e